pkg/setting/setting.go

```diff
@@ -1225,3 +1232,9 @@ func readServerSettings(iniFile *ini.File, cfg *Cfg) error {
 
 	return nil
 }
+
+func readDataSourcesSettings(iniFile *ini.File, cfg *Cfg) error {
+	datasources := iniFile.Section("datasources")
```

Comment:
Shouldn't this be done via cfg.Raw?

```suggestion
	datasources := cfg.Raw.Section("datasources")
```